src/pymor/reductors/aaa.py

```diff
@@ -0,0 +1,417 @@
+# This file is part of the pyMOR project (https://www.pymor.org).
+# Copyright pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (https://opensource.org/licenses/BSD-2-Clause)
+
+import itertools
+
+import numpy as np
+import scipy.linalg as spla
+
+from pymor.core.base import BasicObject
+from pymor.models.transfer_function import TransferFunction
+from pymor.tools.random import new_rng
+
+
+class pAAAReductor(BasicObject):
```

Comment:
I think we always try to use PascalCase for class names. Maybe this one can be PAAAReductor?